thorlcr/activities/iterate/thiterateslave.cpp

```diff
@@ -418,7 +418,7 @@ class CStreamedIteratorSlaveActivity : public CSlaveActivity, public CThorDataLi
     }
     virtual void start()
     {
-        isLocal = container.queryOwnerId() && container.queryOwner().isLocalOnly();
+        bool isLocal = container.queryLocalData() || container.queryOwner().isLocalChild();
```

Comment:
Looks sensible, but member variable 'isLocal', now no longer used and should be removed.